commons/passive/garmin/garmin_generic_heartrate.avsc

```diff
@@ -0,0 +1,43 @@
+{
+   "name":"GarminGenericHeartRate",
+   "type":"record",
+   "namespace":"org.radarcns.passive.garmin",
+   "doc":"Provides the data about real time Heart rate data.",
+   "fields":[
+      {
+         "name":"currentHeartRate",
+         "type":"int",
+         "doc":"Current heart rate."
+      },
+      {
+         "name":"currentRestingHeartRate",
+         "type":"int",
+         "doc":"Current resting heart rate."
+      },
+      {
+         "name":"dailyHighHeartRate",
```

Comment:
Would it make sense to separate the daily data(aggregated I assume) from real time  data?

Reply:
Yes, this is a good suggestion, although I am not sure, how this is calculated and what's the sampling frequency of this. For example, If you get this once a day, then yes it would make sense to separate this but if you get it with each retrieved heart rate data point, then it may be ok to include it here.

Reply:
Daily high hearrate value is received every time the heart rate is received. We will keep this as defined here and not split it.